jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -166,6 +166,10 @@ def user_api(self):
 
         return PluginUserApi(self, expose=expose)
 
+    @property
+    def slice_display_unit_name(self):
+        return 'spectral'
```

Comment:
I actually think this already existed as `slice_axis` (sorry I didn't notice that before!)